style.css

```diff
@@ -71,24 +71,48 @@ header {
     align-items: center;
     justify-content: center;
     margin-top: 200px;
-    border-radius: 25px;  
+    border-radius: 25px; 
+    display: block;
+   
 }
 
 .text h1 {
     margin-top: -50px;
     margin-bottom: 50px;
     background: #56c3e4;
     border-radius: 32px;
+
 }
 
-.features, #container1, #container2, #container3 {
+.features{
     background: rgb(140, 140, 184);  
     border-radius: 50px;
 }
 
-#txt1, #txt2, #txt3 {
-    font-size: 20px;
-    margin-top: -60px;
+
+#container1 {
+    display: flex;
+    margin-top: 40px;
+    background: #4b5768;
+    border: 3px black solid;
+
+}
+
+#container2 {
+    display: flex;
+    margin-top: 40px;
+    background: #3b4c64;
+    border: 3px black solid;
+    
+}
+
+#container3 {
+    display: flex;
+    margin-top: 40px;
+    background: #264570;
+    margin-bottom: 40px;
+    border: 3px black solid;
+    
```

Comment:
The colors need to be uniform for the background of the features, it gives the page a better look. Also you need to consider lighter colors like light shades of gray